cdap-docs/hydrator-manual/source/developing-pipelines.rst

```diff
@@ -407,26 +413,15 @@ stream after performing a projection transformation, you can use a configuration
           },
           {
             "from": "dropProjector",
-            "to": "streamSink"
+            "to": "tableSink"
           }
         ]
       }
     }
 
-A Hydrator real-time pipeline expects an instance property that will create *N* instances
-of the worker that runs concurrently. In Standalone CDAP mode, this is implemented as
-multiple threads; in Distributed CDAP mode, it will create different YARN containers. The
-number of worker instances of a real-time pipeline should not (in general) be changed
-during runtime. If the number of instances needs to be changed, the worker must first be
-stopped, and then the pipeline configuration can be updated to the new number of instances.
-
-The ``instances`` property value needs to be greater than zero. Note that the ``instance``
-property replaces the ``schedule`` property of a Hydrator batch pipeline.
-
 In the example code above, we will use a *ProjectionTransform* (a type of transform) to
-drop and rename selected columns in the incoming data. A *StreamSink* in the final step
-needs a data field property (``body.field``) that it will use as the content for the data
-to be written.
+drop and rename selected columns in the incoming data. A *TableSink* in the final step
+that will write to a CDAP Table.
```

Comment:
Something is missing here; should it be:
```
drop and rename selected columns in the incoming data. A *TableSink* in the final step
requires a schema property that it will use to set the columns and row field for
writing the data to a CDAP Table.